deploy/backplane/srep/hypershift/service-cluster/10-backplane-dev.Role.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: backplane-dev
+  namespace: backplane
+rules:
+# SRE get the backplane dev secret
+# TODO: Remove this when no longer needed
```

Comment:
Explain please why this is needed and when it will no longer be needed.
This should go in the comment and in the referenced JIRA.

Reply:
While I believe we still need that until we get into implement on ISV support for managed openshift,  the answer to 'why' still lies with Haoran and Siu Wa who are on a week long holiday due to chinese new year :')